conf.d/done.fish

```diff
@@ -31,6 +31,20 @@ function __done_get_window_id
 	end
 end
 
+function __done_tmux_active
+	if type -q tmux
+		set tmuxpanes (tmux list-panes -a -F '#{pane_pid}')
+		set mypid %self
+		for tmuxpid in $tmuxpanes
+			if test $tmuxpid -eq $mypid
+				tmux list-panes -F '#{pane_active} #{pane_pid}' | grep "^1 $mypid" > /dev/null
```

Comment:
Why iterate over the available panes? Wouldn't it be enough to return 
`tmux list-panes -F '#{pane_active} #{pane_pid}' | grep "^1 $mypid" > /dev/null` status?